indigo_app/static/javascript/indigo/utils.js

```diff
@@ -15,3 +15,21 @@ $(function() {
 
   $('body').on('click', 'a[data-confirm], button[data-confirm]', handleConfirm);
 });
+
+// Show popover when hovering on selected links
+$('body').on('mouseenter', 'a[data-popup-url]', function() {
+  var _this = this;
+  $.get(this.href + 'popup').then(function(html) {
```

Comment:
This URL should be the contents of the `data-popup-url` attribute.

Reply:
Updated